src/components/Advertisements/Advertisements.test.tsx

```diff
@@ -163,57 +163,6 @@ const ORGANIZATIONS_LIST_MOCK = {
 };
 
 describe('Testing Advertisement Component', () => {
-  test('Testing if the deletion works', async () => {
```

Comment:
Why is this test removed? Is it moved to `AdvertisementEntry.test.tsx`?

Reply:
yes